File: benchmarks/suite5/scan.rs
mod bench_runtime;
use bench_runtime::{suite5_opaque_i64, suite5_print_i64};

fn main() {
    let n: i64 = suite5_opaque_i64(8000000);
    let mut acc: i64 = 0;
    for i in 0..n {
        if i % 3 == 0 || i % 7 == 0 {
            acc += 1;
        }
    }
    suite5_print_i64(acc);
}

File: benchmarks/suite5/scan.go
package main


func main() {
	n := suite5OpaqueI64(8000000)
	var acc int64
	for i := int64(0); i < n; i++ {
		if i%3 == 0 || i%7 == 0 {
			acc++
		}
	}
	suite5PrintI64(acc)
}
